### PR TITLE
Kill Frontend -> Frontend ActiveXML for good!

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

**Comment by ChrisBr** [comment]
Awesome :hankey:

**Comment by hennevogel** [comment]
Marriage done! #450 

Time for Honey :full_moon_with_face: 

:dancers: :dancers: :dancers: :dancers: :dancers: :dancers: :dancers: :dancers: :dancers: :dancers: :dancers: :dancers:

**Comment by mdeniz** [comment]
LGTM :+1:

**Comment by coolo** [comment]
it only took one hackweek and 2 years :)

File: src/api/test/test_helper.rb [hunk 1]
```diff
@@ -294,7 +259,7 @@ def current_user
       Capybara.current_driver = :rack_test
 # crude work around - one day I will dig into why this is necessary
       Minitest::Spec.new('MINE') unless Minitest::Spec.current
-      self.class.start_test_api
+      Suse::Backend.start_test_backend
       #Capybara.current_driver = Capybara.javascript_driver
```
Comment (coolo) [inline src/api/test/test_helper.rb]: you might actually be able to live without it - as the backend class in "the API" is starting the test backend whenever it's needed.

Comment (hennevogel) [inline src/api/test/test_helper.rb]: I've tried it without it, it didn't work, tests where failing all over the place.

**Pull request merged**